src/ol_openedx_chat/static/html/student_view.html

```diff
@@ -1,3 +1,11 @@
-<div>
-      <button type="button" onclick="alert('Hello World!')">Hello World!</button>
+<div class="{{ block_type }}-block-chat-button-container">
+  <div class="ai-chat-button {{ block_type }}-block-chat-button" id="chat-button-{{ block_key }}" data-block-key="{{ block_key }}">
+    {% if block_type == "problem" %}
+      <p>Help me with this problem</p>
+    {% elif block_type == "video" %}
+      <p><b>Questions about this video?</b><br>AskTim</p>
+    {% endif %}
+    <img src="/static/images/icon.svg" alt="Chat Icon" class="chat-icon">
```

Comment:
```suggestion
    <img src="/static/images/icon.svg" alt="" class="chat-icon">
```

The icon is purely decorative. Better for screenreaders to ignore it. The button text is all that screenreaders need.